packages/mocha/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@neutrinojs/mocha",
   "version": "8.2.0",
   "description": "Neutrino preset for testing Neutrino projects with Mocha",
-  "main": "src/index.js",
```

Comment:
Similar to the jest preset case, could the file be left under `src/` for now, so that git knows it's existing content and displays the actual diff? (Happy for the file to be moved either in a PR before this lands, or sometime after)